ecosystem/typescript/sdk_v2/src/internal/account.ts

```diff
@@ -121,3 +147,268 @@ export async function getResource(args: {
   });
   return data;
 }
+
+export async function getAccountTokensCount(args: {
+  aptosConfig: AptosConfig;
+  accountAddress: HexInput;
+}): Promise<GetAccountTokensCountQueryResponse> {
+  const { aptosConfig, accountAddress } = args;
+
+  const address = AccountAddress.fromHexInput({ input: accountAddress }).toString();
+
+  const whereCondition: any = {
+    owner_address: { _eq: address },
+    amount: { _gt: "0" },
+  };
+
+  const { data } = await post<{}, GetAccountTokensCountQuery>({
+    url: aptosConfig.getRequestUrl(AptosApiType.INDEXER),
+    body: {
+      query: GetAccountTokensCount,
+      variables: { where_condition: whereCondition },
+    },
+    originMethod: "getAccountTokensCount",
+    overrides: { WITH_CREDENTIALS: false, ...aptosConfig.clientConfig },
+    isIndexer: true,
+  });
```

Comment:
Let's make this into a helper function, since it looks like you're doing it slightly differently per each call

```
queryIndexer<ReturnType>(
{
  query: Query,
  variables: {},
  name: "QueryName" (Is this required?  What if it differs?  What purpose does it serve?)
  overrides?: {}
}
)

```

We may want to do similarly with the get and post queries that we have for the Fullnode APIs, just to remove duplicate code, and make it easier to read.

Reply:
we should probably have it in the sdk for users to be able to query whatever query they want, might as well add it here